source/whats-new.txt

```diff
@@ -26,6 +27,49 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-4.11:
+
+What's New in 4.11
+------------------
+
+New features of the 4.11 {+driver-short+} release include:
+
+- Recursive Schema Support
+
+  - The driver now supports recursive schemas and provides type safety
+    on dot notation queries up to a depth of 9. Beyond a depth of 9,
```

Comment:
Issue: Per the style guide (Numbers versus Words), for descriptions of quantities less than 10, use spelled-out numbers unless it meets one of the exception cases listed in the aforementioned article.

```suggestion
    for dot notation queries up to a depth of nine. Beyond a depth of nine,
```

Comment applies to all instances in this PR.